src/Notification.ts

```diff
@@ -11,11 +11,11 @@ export class Notification<T> {
   observe(observer: Observer<T>): any {
```

Comment:
Just filed an issue related to this here: https://github.com/Microsoft/TypeScript/issues/6806